packages/design-system-react/src/utils/tw-merge.ts

```diff
@@ -0,0 +1,57 @@
+// utils/tw-merge.ts
+
+import { extendTailwindMerge } from 'tailwind-merge';
+
+// TODO create a test that checks against typographyMap in Text.tsx
```

Comment:
I will do this in https://github.com/MetaMask/metamask-design-system/issues/119

Reply:
Tests for twmerge util here https://github.com/MetaMask/metamask-design-system/pull/122